### DISCUSSION TITLE
Value blank on error when ErrorMessage set in [PostProcessing]

### DISCUSSION THREAD

**Discussion by DrCoolHands** [jomjol/AI-on-the-edge-device#1290] [General]
### The Problem

I have "ErrorMessage" set to "true" in my config.ini however I receive a blank value when there is an error.

The text in the config editor for "ErrorMessage" reads "Do not show error message in return value - in error case, the last valid number will be send out"

curl responce:
```
{
"default":
  {
    "value": "",
    "raw": "05807.25",
    "error": "Rate too high - Read: 5807.25 - Pre: 5607.27",
    "rate": "",
    "timestamp": "2022-11-03T13:24:03"
  }
```

I have tried setting "ErrorMessage" to "false" as well, and also receive a blank value on error.

### Version

12.0.1

### Expected Behavior

With ErrorMessage set to true, the last valid value is expected to be returned. 

### Additional Context

PostProcessing section from config.ini
```
[PostProcessing]
default.DecimalShift = -1
PreValueUse = true
PreValueAgeStartup = 720
AllowNegativeRates = false
default.MaxRateValue = 0.15
default.MaxRateType = AbsoluteChange
;default.ExtendedResolution = false
;default.IgnoreLeadingNaN = false
ErrorMessage = true
CheckDigitIncreaseConsistency = true
```

**Comment by schluo** [comment]
Might be a stupid qestion: But did you reboot the ESP after changing the ini file?

**Comment by DrCoolHands** [comment]
I rebooted after each change and confirmed the restart in the log file.
I have also fully power cycled the ESP with no change to the behaviour.

**Comment by jomjol** [comment]
This is a misunderstanding of the Parameter `ErrorMessage`. This parameter is actually not used anymore, but will be removed.
In case of an detected error, the `value` will be empty and in the field `error`, there will be the information, what went wrong.

Thanks for the hint, I will remove this paramter soon.
